docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md

```diff
@@ -226,3 +226,20 @@ Use this dashboard to:
 - To understand user behavior accessing clusters and servers through Rest API.
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Couchbase-OpenTelemetry/Couchbase-HTTP-Access.png' alt="Access" />
+
+## Create monitors for Couchbase app
+
+import CreateMonitors from '../../../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### Couchbase alerts
+
+| Name | Description | Alert Condition | Recover Condition |
```

Comment:
```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

Reply:
@amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.